docs/build/cpp_package.md

```diff
@@ -0,0 +1,85 @@
+# Building a C++ example
+
+This example shows how to build a C++ project with CMake and use it together with `pixi-build`.
+To read more about how building packages work with pixi see the [Getting Started](./getting_started.md) guide.
+
+We'll start off by creating a project that use [nanobind](https://github.com/wjakob/nanobind) to build Python bindings.
+That we can also test using pixi.
+We'll later combine this example together with a Python package.
+
+## Creating a new project
+
+To get started, create a new project with pixi:
+
+```bash
+pixi init python_bindings
+```
+
+This should give you the basic `pixi.toml` to get started.
+
+We'll now create the following source directory structure:
+```bash
+python_bindings/
+├── CMakeLists.txt
+├── pixi.toml
+└── src
+    └── bindings.cpp
+```
+
+## Creating the project files
+Next up we'll create the:
+  - `pixi.toml` file that will be used to configure pixi.
+  - `CMakeLists.txt` file that will be used to build the bindings.
+  - `src/bindings.cpp` file that will contain the bindings.
```

Comment:
```suggestion

- `pixi.toml` file that will be used to configure pixi.
- `CMakeLists.txt` file that will be used to build the bindings.
- `src/bindings.cpp` file that will contain the bindings.
```

Otherwise the formatting breaks